src/SourceBuild/content/repo-projects/sdk.proj

```diff
@@ -42,6 +42,7 @@
     <RepositoryReference Include="aspnetcore" />
     <RepositoryReference Include="command-line-api" />
     <RepositoryReference Include="deployment-tools" />
+    <RepositoryReference Include="emsdk" Condition="'$(DotNetBuildSourceOnly)' != 'true' and '$(PgoInstrument)' != 'true' and '$(TargetArchitecture)' != 'arm' and '$(TargetArchitecture)' != 'x86'" />
```

Comment:
This is an interesting condition. Can you elaborate on the arch matrix here? Consider adding a comment.

Reply:
there's no technical reason, it's just not worth it to support win-x86 or linux-x86/arm (macOS arm/x86 doesn't exist) given the low amount of users that would run the webassembly build tools there, x64/arm64 covers the vast majority.

I'll add a comment.

Reply:
hm maybe I should switch it around to explicitly check for x64/arm64